news/2021-04-06-mappers-report-march.md

```diff
@@ -0,0 +1,173 @@
+
+We're very excited to see what X a v y will produce in the coming months and hope to see his dedication forge him into a truly excellent mapper!
+
+## <a id="mapping-contests"></a>Mapping Contests
+
+You are up for contests, but you prefer mapping over playing? Then search no more. We have some terrific contests to offer, which still accept participants. All quality-checked and perceived as good!
+
+### Hyperpop Beatmapping Contest
+
+Do you like Hyperpop music? Do you like clicking or placing circles? Then we have a perfect contest for you: [Hyperpop Beatmapping Contest](https://osu.ppy.sh/community/forums/topics/1283577 "Thread")! This osu! mapping contest aims to introduce more mappers to the underrated music genre, Hyperpop. Never heard of it, yet curious? No problem! The registrations are up until the **April 11th**. 
+
+Of course, what could be more exciting than the rewards? The first place gets a shiny new badge for their osu! profile, together with 3 months of osu! supporter. However, the second and third places won't be left out. Their achievement will also be rewarded with osu! supporter.
+
+So what are you waiting for, go and check it out! 
+
+### A Labour of Love: Voting Open
+
+Remember the [Labour of Love](/wiki/Contests/A_Labour_of_Love_Contest) contest? Submissions are in, and now it's on you to pick the winner, but you have only two votes to cast out of eight incredible mapsets. Don't forget that you also have only less than a week to vote!
+
+So make sure to check them all out and vote for your favorite. You can find all the submissions and download them on the [Labour of Love contest page](https://osu.ppy.sh/community/contests/115).
+
+We're excited to see who will take the trophy home!
+
+## <a id="around-community"></a>Around the Community
+
+Last but not least, we want to give other mapping-related projects a voice that does not fit in any of the categories above. All have the same goal: Helping the community. Should you be interested in it, make sure to check them out!
+
+### Community Mentorship Program Spring 2021
+
+The Mentorship program is back with another cycle! Are you interested in mapping but have no idea where to start? Or, are you looking for people to convey your ingenious mapping techniques? Then search no more. The [Community Mentorship Program](https://osu.ppy.sh/community/forums/topics/1273766 "Thread") is exactly what you are looking for.
+
+In this program, experienced mappers will teach people interested in how to create a beatmap. Whether it be osu!, osu!taiko, osu!catch or osu!mania, the program has mapping mentors from every game mode. While the mentor signups have already been closed, the mentee signups will open on **April 19th** and will remain up until **April 25th**. Save the date!
```

Comment:
```suggestion
In this program, experienced mappers are allocated groups of people to tutor on both basic and advanced beatmapping concepts. Whether it be osu!, osu!taiko, osu!catch or osu!mania, the program has mapping mentors from every game mode, and many of the more recently popular mappers have been participants in the program in the past. 

While the mentor signups have already been closed, the mentee signups will open on **April 19th** and will remain up until **April 25th**. Take a look at the project and save the date, it could be your big break into the scene!
```